app/components/Form/RadioButtonGroup.tsx

```diff
@@ -1,11 +1,11 @@
 import { Children, cloneElement } from 'react';
 import styles from './RadioButtonGroup.css';
-import type { Node } from 'react';
+import type { ReactElement } from 'react';
 
 type Props = {
   name: string;
   label: string;
-  children: Node;
+  children: ReactElement | ReactElement[];
 };
```

Comment:
Just an FYI, you can also use `PropsWithChildren` utility type from React for this.

Reply:
Not really, this component can only have `ReactElement`s as children. `ReactNode` or `PropsWithChildren` would allow f.ex. `string`s.